airbyte-webapp/package.json

```diff
@@ -112,7 +112,6 @@
     "@types/json-schema": "^7.0.11",
     "@types/lodash": "^4.14.182",
     "@types/node": "^17.0.40",
-    "@types/query-string": "^6.3.0",
```

Comment:
:information_source: Removed some deprecated package (this was a stub by now, `query-string` include the types itself).